package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.16.5",
```

Comment:
- [ ] Your README file is returning 404 Not Found when clicking on the MIT license link. Kindly add a License file to make your project more professional. 🤝️

![Screenshot from 2023-07-06 12-59-05](https://github.com/tjay1760/Math-Magician/assets/17243325/9306b16d-2a28-4307-997c-1146b799b5fd)